zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -319,6 +319,30 @@ function ParagraphCtrl($scope, $rootScope, $route, $window, $routeParams, $locat
       paragraphText, $scope.paragraph.config, $scope.paragraph.settings.params);
   };
 
+  $scope.runSubsequentParagraphs = function(note,paragraph) {
+    if (!(paragraph.status in ['RUNNING','PENDING']) && paragraph.config.enabled){
+      BootstrapDialog.confirm({
```

Comment:
I personally prefer skip confirmation dialog especially when runSubsequentParagraphs() is called from keyboard shortcut. What do you think? 

And after runSubsequentParagraphs() ran, paragraph lost focus, so need to use mouse again and to focus paragraph.